addon/Feeds/TopSitesFeed.js

```diff
@@ -23,28 +26,27 @@ module.exports = class TopSitesFeed extends Feed {
       // Get metadata from PreviewProvider
       links = yield this.options.getCachedMetadata(links, "TOP_FRECENT_SITES_RESPONSE");
 
+      this.missingData = false;
+
       // Get screenshots if the favicons are too small
       if (experiments.screenshots) {
-        try {
-          links = yield getScreenshots(links, site => {
-            if (site.favicon_height >= 96 && site.favicon_width >= 96 && isRootDomain(site.url)) {
-              // If we are at the "root domain path" and the icon is big enough,
-              // we don't show a screenshot.
-              return false;
-            }
-            return true;
-          });
-          links = links.map(link => {
-            if (link.screenshot) {
+        for (const link of links) {
```

Comment:
you can't use `const` here since it is going to change every loop. Travis caught it:

```
Full Message: SyntaxError: missing = in const declaration

Full Stack: @resource://activity-streams/addon/Feeds/TopSitesFeed.js:33:NaN
```